docs/UserGuide.md

```diff
@@ -154,15 +154,15 @@ Symbol     | Parameter    | Constraints
 Symbol     | Parameter    | Constraints
 -----------|--------------|----------------------------------------------------------------------------------------------------------------------------------------------------------
 **`a`**| `DESCRIPTION` |- Must contain at least 1 alphabetic character and has a limit of 40 characters.
-**`s`**| `START`      |- A singular `DATE-TIME` in the form `DD-MM-YYYY-HH-MM`.
-**`e`**| `END`        |- A singular `DATE-TIME` in the form `DD-MM-YYYY-HH-MM`.
+**`s`**| `START`      |- A singular `DATE-TIME` in the form `DD-MM-YYYY-HH-mm`, where `DD` is the day, `MM` is the month, `YYYY` is the year, `HH` is the hour and `mm` is the minute.
+**`e`**| `END`        |- A singular `DATE-TIME` in the form `DD-MM-YYYY-HH-mm`, where `DD` is the day, `MM` is the month, `YYYY` is the year, `HH` is the hour and `mm` is the minute..
```

Comment:
mm is small letters in the form, but capitalised in the description, perhaps we should standardise to all caps?